Product/SoapUI_Test/ValidationSuite/pom.xml

```diff
@@ -17,11 +17,408 @@
+                                </property>
+                                <property>
+                                    <name>soapui.ext.libraries</name>
+                                    <value>${project.basedir}/src/main/resources/ext</value>
+                                </property>
+                            </soapuiProperties>
+                            <exportAll>true</exportAll>
+                            <printReport>true</printReport>
+                            <junitReport>true</junitReport>
+                            <coverage>false</coverage>
+                            <testFailIgnore>${soapui.testFailIgnore}</testFailIgnore>
+                            <saveAfterRun>false</saveAfterRun>
+                            <host>${host}:${port}</host>
+                            <testSuite>${test.suite}</testSuite>
+                            <testCase>${test.case}</testCase>
+                            <projectProperties>
+                                <value>GatewayPropDir=${project.parent.basedir}/${wildfly.properties.dir}</value>
+                                <value>DBHost=${db.host}</value>
+                                <value>DBPass=${db.password}</value>
+                                <value>DBPort=${db.port}</value>
+                                <value>DBUser=${db.user}</value>
+                            </projectProperties>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>ConnectValidation-soapui-project.xml</id>
+                                <phase>integration-test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <projectFile>${project.basedir}/ConnectValidation-soapui-project.xml</projectFile>
+                                    <outputFolder>${project.build.directory}/soapui/output/ConnectValidation</outputFolder>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>test-verify</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>test-verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <!-- End of soapUI test -->
+                </plugins>
+            </build>
+        </profile>
+    <!-- Old Validation Suite -->
         <profile>
             <id>validation-testing</id>
             <activation>
                 <property>
-                    <name>!validation.skip</name>
+                    <name>validation.skip</name>
```

Comment:
Please rename this to "validation" or "validation.do".  Leaving it as "validation.skip" for an activation property is confusing.